download.py

```diff
@@ -199,7 +208,7 @@ def check_gecko_driver():
     with open('ustvgo.m3u8', 'w') as file:
         file.write('#EXTM3U\n\n')
         for name, url in video_links:
-            file.write('#EXTINF:-1,' + name + '\n')
+            file.write('#EXTINF:-1 tvg-id="' + channel_id_override[name] + '" ' + 'group-title="' + channel_categories[name] + '", ' + name + '\n')
```

Comment:
You better get dictionary values with ```.get(key, default_value)``` to avoid ```KeyError```'s in case ```ustvgo``` would change their channel names.